packages/server-core/src/media/storageprovider/s3.storage.ts

```diff
@@ -664,6 +664,38 @@ export class S3Provider implements StorageProviderInterface {
     )
   }
 
+  /**
+   * Get the folder Hiearchy
+   * @param folderName Name of folder in storage
+   * @param depth Get all folder by the depth
+   */
+  async getFolderHierarchy(folderName: string, depth) {
```

Comment:
is this funciton used anywhere? If not, please remove it, and if so, please implement the corresponding function in the storage provider interface and local storage provider with appropriate tests